src/test/resources/net/sf/jabref/exporter/MsBibExportFormatTest1.bib

```diff
@@ -0,0 +1,44 @@
+@unpublished{,
+  author =                      {Hanns-Jörg Stoß},
+  month =                       {July},
+  note =                        {a Note},
+  abstract = {there is something described about turing machines. They have k bands but still are as strong as turing machines that have just one. This will be proofed.},
```

Comment:
This does not always hold true. Often, when you publish something, you have to sign a lot of paperwork which may influence your rights to publish parts of it anywhere. Please replace this with dummy text, as we do not want to keep track which abstracts in our test data are OK to be included and which are not.